### PR TITLE
[BLOCKSCOUT-CLIENT] New codegenerated client!

### PR DESCRIPTION
**Pull request by sevenzing**

### CODE REVIEW

File: libs/blockscout-client/crate/.travis.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+language: rust
```
Comment (bragov4ik) [inline libs/blockscout-client/crate/.travis.yml:1]: do we use travis somewhere? 🤔

Comment (sevenzing) [reply libs/blockscout-client/crate/.travis.yml:1]: nope, will delete it

File: libs/blockscout-client/crate/src/apis/blocks_api.rs [hunk 2]
```diff
@@ -0,0 +1,209 @@
+/*
+ * BlockScout API
+ *
+ * API for BlockScout web app
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * Contact: you@your-company.com
```
Comment (bragov4ik) [inline libs/blockscout-client/crate/src/apis/blocks_api.rs:7]: probably it should be set to some blockscout email (or removed)? nit

Comment (sevenzing) [reply libs/blockscout-client/crate/src/apis/blocks_api.rs:7]: I didnt file a way to indicate email in config for [openapi-generator](https://github.com/OpenAPITools/openapi-generator)

**Review by rimrakhimov — APPROVED** [review]

**Pull request merged**